compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -66,11 +66,11 @@ struct ROCmOptions {
   void bindOptions(OptionsBinder &binder) {
     using namespace llvm;
     static cl::OptionCategory category("ROCm HAL Target");
-    binder.opt<std::string>("iree-rocm-target-chip", targetChip,
-                            cl::cat(category), cl::desc("ROCm target chip."));
+    binder.opt<std::string>("iree-hip-target", target, cl::cat(category),
+                            cl::desc("HIP LLVM target."));
```

Comment:
same comment - put an example, a list of possible values, a link to documentation, etc. here